config.py

```diff
@@ -43,6 +43,11 @@
 APP_INFO_SQLITE_FILE = 'sqlite:///static_data/app-info.db' + \
     ("~test" if TEST else "")
 
+# IOC stalkware indicators
+IOC_PATH = "data/stalkerware-indicators/"
+IOC_FILE = IOC_PATH + "ioc.yaml"
+
+
 # we will resolve the database path using an absolute path to __FILE__ because
 # there are a couple of sources of truth that may disagree with their "path
 # relavitity". Needless to say, FIXME
```

Comment:
There’s likely a way to do this using github `$RUNNER` environment variables to get to specific paths

Reply:
yeah, i used `config.py` as most of the config variables are stored in this. will have to eventually migrate to env variables in a separate commit